script/seed.js

```diff
@@ -192,11 +188,11 @@ async function seed() {
       john: users[4],
     },
     addresses: {
-      address1: addresses[0], 
-      address2: addresses[1], 
-      address3: addresses[2], 
-      addresses4: addresses[3], 
-      addresses5: addresses[4]
+      address1: addresses[0],
+      address2: addresses[1],
+      address3: addresses[2],
+      addresses4: addresses[3],
```

Comment:
Why change key from "address" to "addresses"?

Reply:
I didn't change anything here, it's just the code editor cleaned up the spacing.